data/app.rednotebook.rednotebook.appdata.xml

```diff
@@ -0,0 +1,89 @@
+  <url type="translate">https://translations.launchpad.net/rednotebook</url>
+
+  <!-- https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#tag-dapp-screenshots -->
+  <screenshots>
+    <screenshot type="default">
+      <caption>RedNotebook - Edit mode</caption>
+      <image>https://rednotebook.app/screenshots/rednotebook-2.0-1.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>RedNotebook - Preview mode</caption>
+      <image>https://rednotebook.app/screenshots/rednotebook-2.0-2.png</image>
+    </screenshot>
+  </screenshots>
+
+  <!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-content_rating -->
+  <content_rating type="oars-1.0" />
+
+  <!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-translation -->
+  <translation type="gettext">rednotebook</translation>
+
+  <!-- https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#tag-dapp-launchable -->
+  <launchable type="desktop-id">app.rednotebook.rednotebook.desktop</launchable>
+
+  <!-- https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#tag-dapp-provides -->
+  <provides>
+    <binary>rednotebook</binary>
+  </provides>
+
+  <!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-releases -->
+  <releases>
+    <release version="2.22" date="2021-04-25"/>
+    <release version="2.21" date="2020-12-07"/>
+    <release version="2.20" date="2020-08-03"/>
+    <release version="2.19" date="2020-05-04"/>
+    <release version="2.18" date="2020-02-29"/>
+    <release version="2.17" date="2020-02-23"/>
+    <release version="2.16" date="2020-01-23"/>
+    <release version="2.15" date="2019-12-04"/>
+    <release version="2.14" date="2019-11-17"/>
+    <release version="2.13" date="2019-11-07"/>
+    <release version="2.12" date="2019-11-02"/>
+    <release version="2.11.1" date="2019-04-07"/>
+    <release version="2.11" date="2019-03-26"/>
+    <release version="2.10" date="2019-03-24"/>
+    <release version="2.9.1" date="2019-03-20"/>
+    <release version="2.9" date="2019-03-17"/>
+    <release version="2.8.1" date="2019-03-17"/>
+    <release version="2.8" date="2018-11-15"/>
+    <release version="2.7.1" date="2018-11-13"/>
+    <release version="2.7" date="2018-11-06"/>
+    <release version="2.6.1" date="2018-08-21"/>
+    <release version="2.6" date="2018-08-20"/>
+    <release version="2.5" date="2018-06-08"/>
+    <release version="2.4" date="2018-03-07"/>
+    <release version="2.3" date="2017-09-25"/>
+    <release version="2.2" date="2017-09-08"/>
+    <release version="2.1.5" date="2017-08-09"/>
+    <release version="2.1.4" date="2017-08-08"/>
+    <release version="2.1.3" date="2017-08-06"/>
+    <release version="2.1.2" date="2017-08-03"/>
+    <release version="2.1.1" date="2017-08-02"/>
+    <release version="2.1" date="2017-07-29"/>
+    <release version="2.0" date="2017-05-19"/>
```

Comment:
Nice work collecting these!

Reply:
Cannot take credit as CHANGELOG.md gave me all the info I needed.